### ISSUE TITLE
Ensure that no components rely on ngAria ng-model tabindex behavior

### ISSUE THREAD

**Issue by jelbourn** [angular/material#3330]
By default, ngAria will add tabindex to things with ng-model. This is not always wanted, as it's fine to have a directive that has some model value but isn't a tab stop (and instead has children that are tab stops). Because of this, if a user turns that behavior off with $ariaProvider, none of the material components should being missing their expected tabindex.

This issue is to double-check that none of the material components are in fact depending on this behavior.

**Comment by marcysutton** [comment]
:thumbsup:
